cerberus/tests/test_validation.py

```diff
@@ -808,6 +808,18 @@ def test_dependencies_dict_with_required_field():
     assert_success({'foo': 'bar', 'bar': 'foo'}, schema)
 
 
+def test_dependencies_dict_with_nullable_fields():
```

Comment:
please test the `nullable` issue in an extra test and add a reference to #305 as URL here.